calm/app/models/announcement.rb

```diff
@@ -0,0 +1,10 @@
+class Announcement < ActiveRecord::Base
+    def self.all_ratings
```

Comment:
I didn't remove that function because it could be used in the future. For instance, we may change the name 'rating' to 'tag' or something, then that function could easily be modified to get all tags.